diff -rupN busybee-0.5.2/configure.ac busybee-0.5.2-new/configure.ac
--- busybee-0.5.2/configure.ac	2014-10-28 11:22:48.728788529 +0800
+++ busybee-0.5.2-new/configure.ac	2014-10-28 11:21:20.728792594 +0800
@@ -19,8 +19,8 @@ AC_LANG(C++)
 ANAL_WARNINGS
 
 # Checks for libraries.
-PKG_CHECK_MODULES([PO6], [libpo6 >= 0.5])
-PKG_CHECK_MODULES([E], [libe >= 0.7])
+#PKG_CHECK_MODULES([PO6], [libpo6 >= 0.5])
+#PKG_CHECK_MODULES([E], [libe >= 0.7])
 
 # Checks for header files.
 AC_CHECK_HEADERS([netinet/in.h stdint.h stdlib.h sys/socket.h unistd.h])
